### PR TITLE
Document general infrastructure set up and how to get logs

### PR DESCRIPTION
**Pull request by tahb**
# Context

# Changes in this PR

After running through this myself and recently with @bryangaledxw I think we know enough to commit our current understanding.

We could put this in Confluence but I decided to keep this as close to the repository as possible for better accessibility.

# Release checklist

As part of our continuous deployment strategy we must ensure that this work is
ready to be released at any point. Before merging to `main` we must first
confirm:

## Pre merge checklist

- [ ] Have all changes to any dependencies been deployed to both `preprod` and
    `production` in a backwards compatible way? (This includes our API,
    infrastructure, third-party integrations and any secrets or environment variables)
- [ ] Has a required data migration been included in this change or been done in
    advance?

## Post merge checklist

Once we've merged we now need to release this through to production before
considering it done.

[Find the build-and-deploy job in CircleCI](https://app.circleci.com/pipelines/github/ministryofjustice/hmpps-temporary-accommodation-ui)
and work through the following steps:

- [ ] Has the product manager asked to provide approval for this feature?
  - [ ] Has the product manager approved?
- [ ] Manually approve release to preprod
- [ ] Verify change released to preprod
- [ ] Manually approve release to prod
- [ ] Verify change released to production

Should a release fail at any step, you as the author should now lead the work to
fix it as soon as possible.

### CODE REVIEW

**Comment by tahb** [comment]
@pezholio @edavey FYI incase you want to copy any of this for AP UI. I'll propose one for the API.

File: doc/architecture/decisions/how-to/manage-infrastructure.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+kubectl --namespace hmpps-community-accommodation-<env> get pods
+```
+
+Follow the logs:
+
+```bash
+kubectl -n hmpps-community-accommodation-<env> logs --follow <pod name> --all-containers
+```
+
+### View/change an environment variable
```
Comment (bryangaledxw) [inline doc/architecture/decisions/how-to/manage-infrastructure.md]: Does this need any caveating? Surely the "proper" way to change environment variables is either though changes to the namespace in cloud-platform, or changing our helm config?

Comment (tahb) [reply doc/architecture/decisions/how-to/manage-infrastructure.md]: Ah ok, I think I should clarify that this is about setting/changing the _values_ of _existing_ variables, and not for defining new environment variables. [Setting variables and giving them dummy values is indeed done via Helm](https://github.com/ministryofjustice/hmpps-temporary-accommodation-ui/blob/main/helm_deploy/hmpps-temporary-accommodation-ui/values.yaml#L50).

Comment (tahb) [reply doc/architecture/decisions/how-to/manage-infrastructure.md]: Updated!

**Review by bryangaledxw — APPROVED** [review]
LGTM

**Pull request merged**